src/social-network-provider/twitter.com/ui/injectPostInspector.tsx

```diff
@@ -1,17 +1,42 @@
 import type { PostInfo } from '../../../social-network/ui'
 import { injectPostInspectorDefault } from '../../../social-network/defaults/injectPostInspector'
+import { twitterEncoding } from '../encoding'
 
 export function injectPostInspectorAtTwitter(current: PostInfo) {
     return injectPostInspectorDefault({
         zipPost(node) {
             const content = node.current.parentElement?.querySelector<HTMLDivElement>('[lang]')
 
             if (content) {
-                content.style.maxHeight = '1.5em'
-                content.style.whiteSpace = 'nowrap'
-                content.style.overflow = 'hidden'
-                content.style.textOverflow = 'ellipsis'
+                for (const a of content.querySelectorAll('a')) {
+                    if (twitterEncoding.payloadDecoder(a.title)) hideDOM(a)
+                    if (a.title === 'http://maskbook.com') hideDOM(a)
+                }
+                for (const span of content.querySelectorAll('span')) {
+                    // match (.) (\n) (—§—) (any space) (/*)
+                    // Note: In Chinese we can't hide dom because "解密这条推文。\n—§—" is in the same DOM
+                    // hide it will break the sentence.
+                    if (span.innerText.match(/^\.\n—§— +\/\* $/)) hideDOM(span)
+                    // match (any space) (*/) (any space)
+                    if (span.innerText.match(/^ +\*\/ ?$/)) hideDOM(span)
+                }
+
+                const parent = content.parentElement?.nextElementSibling as HTMLElement
+                if (parent && matches(parent.innerText)) {
+                    parent.style.height = '0'
+                    parent.style.overflow = 'hidden'
+                }
             }
         },
     })(current)
 }
+function matches(str: string) {
+    return str.includes('maskbook.com') && str.includes('Make Privacy Protected Again')
+}
```

Comment:
```suggestion
function matches(input: string) {
    return /maskbook\.com/i.test(input) && /Make Privacy Protected Again/i.test(input)
}
```

use regex pattern ignore case sensitive